include/sexp/sexp-error.h

```diff
@@ -40,25 +40,31 @@ class sexp_exception_t : public std::exception {
     enum severity { error = 0, warning = 1 };
 
   protected:
-    static severity verbosity_;
-    static bool     interactive_;
+    static severity vrb;
```

Comment:
Do we really need to make these renames? It's more readable with full names with underscore attached, as for me.

Reply:
I did full names using but using other approach. Sorry, underscopes at the end harm my aestetic senses

Reply:
@maxirmx This is actually recommended by Google C++ style guide: https://google.github.io/styleguide/cppguide.html#Variable_Names
Anyway, it's up to you.